Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -532,15 +532,26 @@ export const drawRoute = function (data) {
                 return function (sourceName) {
                   return function (destinationName) {
                     return function (specialLocation) {
-                      return function () {
-                        console.log("I AM HERE ------------------ IN DRAW ROUTE");
-                        try {
-                          return window.JBridge.drawRoute(JSON.stringify(data), style, trackColor, isActual, sourceMarker, destMarker, polylineWidth, type, sourceName, destinationName, JSON.stringify(specialLocation));
-                        } catch (err) {
-                          console.log("Catch error" + err);
-                          return window.JBridge.drawRoute(JSON.stringify(data), style, trackColor, isActual, sourceMarker, destMarker, polylineWidth, type, sourceName, destinationName);
-                        }
-                      };
+                      return function (isAnimation){
+                        return function () {
+                          console.log("I AM HERE ------------------ IN DRAW ROUTE");
+                          try {
+                            return window.JBridge.drawRoute(JSON.stringify(data), style, trackColor, isActual, sourceMarker, destMarker, polylineWidth, type, sourceName, destinationName, JSON.stringify(specialLocation), isAnimation);
```

Comment:
@S-PRAVINKUMAR is the same change done in IOS ?

Reply:
Yes